app/src/components/widgets/multiLevelWidgets/cumulativeTrendChart/cumulativeTrendChart.jsx

```diff
@@ -1,17 +1,30 @@
-import { PureComponent, Fragment } from 'react';
+import { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import isEqual from 'fast-deep-equal';
+import { connect } from 'react-redux';
 import { injectIntl, intlShape } from 'react-intl';
 import { NoDataAvailable } from 'components/widgets';
 import { VirtualPopup } from 'components/main/virtualPopup';
 import { ChartJS } from 'components/widgets/common/chartjs';
 import { ActionsPopup } from 'components/widgets/common/actionsPopup';
+import {
+  defectLinkSelector,
+  statisticsLinkSelector,
+  TEST_ITEMS_TYPE_LIST,
+  DEFAULT_LAUNCHES_LIMIT,
+} from 'controllers/testItem';
+import { defectTypesSelector } from 'controllers/project';
+import { activeProjectSelector } from 'controllers/user';
+import { TEST_ITEM_PAGE } from 'controllers/pages';
+import { PASSED, FAILED, SKIPPED, INTERRUPTED, IN_PROGRESS } from 'common/constants/testStatuses';
+import { formatAttribute } from 'common/utils/attributeUtils';
 import SearchIcon from 'common/img/search-icon-inline.svg';
 import FiltersIcon from 'common/img/filters-icon-inline.svg';
+import { DEFECTS, TOTAL_KEY } from '../../common/constants';
 import { getChartData } from './chartjsConfig';
 import { CumulativeChartLegend } from './legend/cumulativeChartLegend';
-import { CumulativeDetails } from './cumulativeDetails';
```

Comment:
Please remove if the component not needed. Also please delete all related unused components,